src/iuse.cpp

```diff
@@ -8347,8 +8347,7 @@ std::optional<int> iuse::craft( Character *p, item *it, const tripoint & )
         return std::nullopt;
     }
     const recipe &rec = it->get_making();
-    const inventory &inv = p->crafting_inventory();
-    if( !p->has_recipe( &rec, inv, p->get_crafting_group() ) ) {
+    if( !p->has_recipe( &rec ) ) {
         p->add_msg_player_or_npc(
```

Comment:
Same as above.

+ Maybe it should differentiate between '"Doesn't know recipe" and "Doesn't have the skills."

Reply:
Yup true too

Reply:
Can do on master d30757ea5f too...

Reply:
current status on my local branch: **fixed**, but there is no feedback, so **needs further work**

Reply:
Decided against fixing it here.